src/bundle/Resources/config/deprecated_routing_commerce_legacy_commerce.yaml

```
@@ -0,0 +1,392 @@
+ibexa.compatibility_layer.content:
+    resource: 'deprecated_routing_experience_legacy_commerce.yaml.yaml'
+
+silver_eshop_rest.dashboard_charts:
+    alias: ibexa.commerce.rest.dashboard.charts
+    deprecated:
```

We discussed deprecation standards and we decided to follow https://github.com/ibexa/engineering/wiki/PHP-Conventions%3A-Deprecation-standards#symfony-services-deprecation

---

This will produce the following message "Since ibexa/commerce-admin-ui 4.0: The "
silver_eshop_rest.dashboard_charts" route alias is deprecated. You should stop using it, as it will be removed in the future.". 

Routing deprecation is not covered in linked document so the question is if we are ok with message above or should we customize it ?

---

We're missing the removal deadline, and otherwise we provide a similar message.

```
Since ibexa/commerce-admin-ui 4.0: The "
silver_eshop_rest.dashboard_charts" route alias is deprecated. You should stop using it, as it will be removed in 5.0.
```
?

```suggestion
    deprecated:
        message: 'Since ibexa/commerce-admin-ui 4.0: The "
%alias_id%" route alias is deprecated. You should stop using it, as it will be removed in the future.'
```

---

Fixed in https://github.com/ibexa/compatibility-layer/pull/50/commits/aad4da2924f962ddafb1131417834193e81d9346